pkg/scaffold/api.go

```diff
@@ -194,6 +190,15 @@ func (api *API) scaffoldV2() error {
 		}
 		fmt.Println(path)
 
+		scaffold := &Scaffold{
+			Plugins: api.Plugins,
+		}
+
+		universe, err := api.buildUniverse(r)
+		if err != nil {
+			return fmt.Errorf("error scaffolding APIs: %v", err)
```

Comment:
A different error message differentiates this return from that under `scaffold.Execute`:
```suggestion
			return fmt.Errorf("error building API scaffold: %v", err)
```